source/code-generation/query-converter/bulk-conversion.txt

```diff
@@ -0,0 +1,104 @@
+.. _rm-bulk-conversion:
+
+===============
+Bulk Conversion
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Bulk conversion creates a job that queues and executes each query 
+conversion from a single request. You can select and convert one 
+or more queries, stored procedures, triggers, and views in bulk 
+by enabling the multi-select toggle. 
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
```

Comment:
I know this isn't part of this PR, but the max query size is now 40,000 characters, not 10,000.